app/models/answer.rb

```diff
@@ -186,6 +181,10 @@ def should_validate?(field)
     end
   end
 
+  def simple_location_answer?
+    qtype.name == 'location' && !value.blank?
+  end
```

Comment:
True! Thanks for pointing it out! I guess I just extracted the old code to DRY it up but didn't really look into it.